<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPrintTables</class>
 <widget class="QDialog" name="DialogPrintTables">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>הדפסת לוחות</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>אפשרויות הדפסת לוח</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="comboBoxType">
          <item>
           <property name="text">
            <string>לוח לשנה , לשנת</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>לוח לחודש, בשנת</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxYear"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>מחודש</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxFromMonth">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>עד</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxToMonth">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>חודש אחד בלבד יודפס אם שני תיבות החודש יהיו שוות.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxIncludeEvents">
        <property name="text">
         <string>כלול ארועים מהלוח</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
